docs/content/en/docs/operator-manual/piped/installation/installing-on-cloudrun.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Installing on CloudRun"
+linkTitle: "Installing on CloudRun"
+weight: 2
+description: >
+  This page describes how to install Piped on CloudRun.
+---
+
+## Prerequisites
+
+##### Having piped's ID and Key strings
+- Ensure that the `piped` has been registered and you are having its PIPED_ID and PIPED_KEY strings.
+- If you are not having them, this [page](/docs/operator-manual/control-plane/registering-a-piped/) guides you how to register a new one.
+
+##### Preparing SSH key
+- If your Git repositories are private, `piped` requires a private SSH key to access those repositories.
+- Please checkout [this documentation](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for how to generate a new SSH key pair. Then add the public key to your repositories. (If you are using GitHub, you can add it to Deploy Keys at the repository's Settings page.)
+
+## Installation
+
+- Preparing a piped configuration file as the following:
+
+  ``` yaml
+  apiVersion: pipecd.dev/v1beta1
+  kind: Piped
+  spec:
+    projectID: {PROJECT_ID}
+    pipedID: {PIPED_ID}
+    pipedKeyData: {BASE64_ENCODED_PIPED_KEY}
+    webAddress: {CONTROL_PLANE_WEB_ADDRESS}
+    # Write in a format like "host:443" because the communication is done via gRPC.
+    apiAddress: {CONTROL_PLANE_API_ADDRESS}
+
+    git:
+      sshKeyData: {BASE64_ENCODED_PRIVATE_SSH_KEY}
+
+    repositories:
+      - repoId: {REPO_ID_OR_NAME}
+        remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+        branch: {GIT_BRANCH}
+
+    # Optional
+    # Enable this Piped to handle CLOUD_RUN application.
+    cloudProviders:
+      - name: cloudrun-in-project
+        type: CLOUDRUN
+        config:
+          project: {GCP_PROJECT_ID}
+          region: {GCP_PROJECT_REGION}
+
+    # Optional
+    # Uncomment this if you want to enable this Piped to handle Terraform application.
+    #  - name: terraform-gcp
+    #    type: TERRAFORM
+
+    # Optional
+    # Uncomment this if you want to enable SecretManagement feature.
+    # https://pipecd.dev/docs/user-guide/secret-management
+    # secretManagement:
+    #   type: KEY_PAIR
+    #   config:
+    #     privateKeyData: {BASE64_ENCODED_PRIVATE_KEY}
+    #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+  ```
+
+- Creating a new secret in [SecretManager](https://cloud.google.com/secret-manager/docs/creating-and-accessing-secrets) to store above configuration data securely
+
+``` console
+gcloud secrets create cloudrun-piped-config --data-file={PATH_TO_CONFIG_FILE}
+```
+
+- Running Piped in CloudRun
+
```

Comment:
nits, how about adding a simple note here: 
```
Prepare service config for your piped as below
Note: Fields which set to '1' are strict to be set with that value to ensure piped work correctly.
```

Reply:
Nice. Let me apply this.

Reply:
Done!